#position
float64 x  # x-coordinate of the robot
float64 y  # y-coordinate of the robot
#velocity
float64 vel_x  # x-velocity of the robot
float64 vel_y  # y-velocity of the robot

# custom message published by the action client
